salesagility/SuiteCRM
# Remove XML_HTMLSax3

## Description

Part of #7605.

This was only ever used as part of HTMLPurifier (as an experimental parser), as far as I can tell. Since HTMLPurifier is no longer vendored in the repository, this library no longer needs to be vendored either.

You can see where it was used in the vendored HTMLPurifier library on hotfix-7.10.x [here](https://github.com/salesagility/SuiteCRM/blob/de09d4e362e6d8827e224e3cfd290d40a9b61dcc/include/HTMLPurifier/standalone/HTMLPurifier/Lexer/PEARSax3.php). That file no longer exists in the repo on develop, so I think we can safely remove it.

## How To Test This
Make sure the tests pass, you can search the repo for references to XML_HTMLSax3 to verify that there's no remaining usage of it.

## Types of changes

### Final checklist
